src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -76,19 +76,23 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
         {
             var files = new List<FileIdentifier>();
 
-            foreach (ILibraryInstallationState state in manifest.Libraries.Where(l => l.IsValid(out var errors)))
+            if (manifest != null)
             {
-                IEnumerable<FileIdentifier> stateFiles = await GetFilesWithVersionsAsync(state).ConfigureAwait(false);
-
-                foreach (FileIdentifier fileIdentifier in stateFiles)
+                foreach (ILibraryInstallationState state in manifest.Libraries.Where(l => l.IsValid(out var errors)))
```

Comment:
nit: var?

Reply:
For that matter, if you're not using errors, just do `out _` instead.

Reply:
This code was there before. We have a bug logged to review use of var throughout LibMan code.